src/spring-cloud/azext_spring_cloud/buildpacks_binding.py

```diff
@@ -0,0 +1,80 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=wrong-import-order
+import shlex
+from ._enterprise import DEFAULT_BUILD_SERVICE_NAME
+from ._exception_enterprise import BuildpacksBindingOperationError
+from .vendored_sdks.appplatform.v2022_05_01_preview import models
+
+
+def buildpacks_binding_create(cmd, client, resource_group, service,
+                              name, type, properties=None, secrets=None):
+    _assert_binding_not_exists(client, resource_group, service, name)
```

Comment:
nit: You can move the ensure exist and not exist to validator and then merge `buildpacks_binding_create` and `buildpacks_binding_set`

Reply:
Great idea. Refined